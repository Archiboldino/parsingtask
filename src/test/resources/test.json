[{"r030":986,"txt":"Бразильський реал","rate":7.005109,"cc":"BRL","exchangedate":"10.06.2018"},{"r030":972,"txt":"Сомонi","rate":2.897071,"cc":"TJS","exchangedate":"10.06.2018"}]