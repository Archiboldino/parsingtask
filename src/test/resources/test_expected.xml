<?xml version="1.0" encoding="UTF-8"?>
<catalog>
  <notebook>
    <person id="1">
      <name>asd</name>
      <address>123</address>
      <cash>123</cash>
      <education>sa</education>
    </person>
    <person id="2">
      <name>dsa</name>
      <address>addr</address>
      <cash>1000000</cash>
      <education>as</education>
    </person>
  </notebook>
</catalog>
